ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -18,6 +18,7 @@ aggregation:
   min: "Minimum value (e.g. Min_Ventilation_Air_Flow_Setpoint)"
   total: "Sum total of some set of values (e.g. total_request_heating_count)"
 component:
+  ballast: "a type of electrical resistor or reactance, fixed in-line with the circuits that supply mains power to a light bulb."
```

Comment:
Proposed, more informative definition: "An electrical ballast is a device placed in series with a load to limit the amount of current in an electrical circuit. Commonly a component of light fixtures."